rsts/concepts/execution_timeline.rst

```diff
@@ -5,3 +5,62 @@ Timeline of a workflow execution
 ########################################
 
 .. image:: https://raw.githubusercontent.com/lyft/flyte/assets/img/flyte_wf_timeline.svg?sanitize=true
+
+
+The illustration above refers to a simple workflow, with 2 nodes N1 & N2. This can be represented as follows,
+
+.. mermaid::
+
+    graph LR;
+         Start --> N1;
+         N1 --> N2;
+         N2 --> End;
+
+
+Acceptance
+===========
+Every Workflow starts in the ``Acceptance`` phase. Acceptance refers to the time it takes for the workflow to start execution in FlytePropeller from the time the FlyteAdmin service receives a request.
+Usually in this section the K8s queuing latency is the highest component. But overall Acceptance latency of <5s is desirable.
+
+**This is one of the latencies that the core team is working on optimizing**
+
+Transition
+===========
+Transition latency refers to the time between success node executions, i.e between N1 & N2. For the first node this latency encapsulate executing the ``Start node`` as well. For the last node to End node, this encapsulates executing ``End node`` as well.
+the latency involves, time it takes to,
+#. Gather outputs for a node after the node completes
+#. Send an observation event to FlyteAdmin. Failing to do so is regarded as an error and will be tried untill successful or we exhaust system max retries.
+#. Persist data to Kubernetes
```

Comment:
This refers to the CRD updates for phase changes correct?

Reply:
yup